### DISCUSSION TITLE
По-русски: система начисления опыта.

### DISCUSSION THREAD

**Discussion by sergunadze** [codewars/codewars.com#2910] [General]
Я тут "новый". Возможно это уже обсасывалось сто раз. Вот я имею базовые знания и решаю легкие задачки, но решаю их много. В итоге у меня рейтиг вырос с 8 до 6. При этом я 100% не смогу решить задачу с рейтингом 6. Считаю данное начисление опыта не верным. Если решаешь только задачи уровня 8, то дальше уровня 8 тебе делать нечего.

**Comment by Blind4Basics** [comment]
Hello,

- In english, please (even if you need to use an automatic translator to do so.
- do you have an actual question, there? (maybe my translator missed it... x) )

---

Привет,

- На английском, пожалуйста (даже если для этого нужно использовать автоматический переводчик.
- У вас есть актуальный вопрос? (может быть, мой переводчик пропустил это... х))

**Reply by sergunadze** [reply]
I'm "new" here. Perhaps it has already been sucked a hundred times. Here I have basic knowledge and solve easy problems, but I solve a lot of them. As a result, my rating increased from 8 to 6. At the same time, I will not be able to solve the problem with a rating of 6 100%. I consider this accrual of experience to be incorrect. If you solve only level 8 problems, then you have nothing to do beyond level 8.

**Comment by JohanWiltink** [comment]
Kata rank and solver rank do not correspond. It is unfortunate that both are expressed in the same units ( kyu ).

**Comment by VasilyZhiganov** [comment]
He means that when solving problems of level 8, the account is upgraded and then there are tasks that cannot be solved for his skills.

Personally, I think that it would be cool if the increase in the account level would be at the request of the user himself and solve one of the tasks, the future level.
For example, to move from level 8 to level 7, you should solve the problem of level 7 and so on.
And until this moment, the user is allowed to solve problems only only at his level.

**Reply by hobovsky** [reply]
> And until this moment, the user is allowed to solve problems only only at his level.

But users are allowed to solve any kata they want, any level they want? The kata search page allows for arbitrary filtering and if a user wants to solve only 8 kyu tasks, they can filter for 8 kyu tasks and solve only them, no matter the user's rank.
